gammapy/modeling/models/spatial.py

```diff
@@ -612,6 +654,55 @@ def evaluation_region(self):
         """Evaluation region consistent with evaluation radius"""
         return self.to_region(x_sigma=5)
 
+    def plot_extension_error(self, ax=None, x_sigma=1.5, **kwargs):
```

Comment:
Most of the plotting code is duplicated, only the region definition differs from one model to the other.

This could be simplified by putting most of the plotting code in the base `SpatialModel` and create e.g. hidden `GaussianSpatialModel._region_error` that will return the error ellipse (or maybe just its parameters).